app/models/user.rb

```diff
@@ -8,5 +8,6 @@ class User < ApplicationRecord
 
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :rents, dependent: :restrict_with_exception
```

Comment:
this looks like belongs to another PR 🤔

Reply:
nope, model rents was implemeted before this pr